GOVERNANCE.md

```diff
@@ -1,45 +1,120 @@
+
+#### Working Groups
+
+A Working Group is formed by community developers who gather together to accomplish a specific goal. To achieve the goal, some WGs may span over multiple SIGs, and some WGs may only focus on something specific in a specific SIG.
+
+Each WG has its life cycle. Once the goal is completed, the group will be disbanded. The only goal of WG operations and management is to ensure that the goals set by the group are completed at the right time. In general, the working group hold periodic meetings to summarize the current project progress and determine the plans for next steps.
+
+<!---to add Working Group governance link -->
+
+## Contributions
+
+Anyone can contribute to the project, regardless of their skills, as there are many ways to contribute. The various ways of contributing are described in more detail in a separate document. For details, see [Contributing to TiKV](https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md).
+
+## Decision making and voting
+
+Decisions are build on consensus between PMC members or Maintainers. Depending on the nature, decisions about the future of the project are made by the PMC or by Maintainers. As the technical authority, Maintainers set technical direction and priorities for the sub-project, while PMC make other high-level decisions like establishment and operation policies of sub-projects, sub-structures,promoting new Maintainers, security processes, etc.
+
+New proposals and ideas (changes of major features, organization, or processes) can be brought to the PMC or Maintainers’ attention through the [Request for Comments (RFC)](https://github.com/tikv/rfcs) process. For the change to happen, the RFC must earn the supermajority (2/3) votes in the corresponding group.
+
+## Conflict resolution
+
+In general, we prefer that technical issues and maintainer membership are amicably worked out between the persons involved. If a dispute cannot be decided independently, depending on the nature, the PMC or Maintainers can be called in to resolve the issue by voting. The same PR can be used or a separate PR can be opened in the concerned repository for voting.
+
+## Adding new projects to the TiKV GitHub organization
+
+New projects can be added to the TiKV organization via GitHub issue discussion in one of the existing projects in the organization, as long as as they adheres to the [CNCF charter](https://www.cncf.io/about/charter/) and the guidelines in this governance. Once sufficient discussions have taken place (normally 3-5 business days but also depending on the volume of conversation), the Maintainers of the project where the issue was opened (since different projects in the organization may have different maintainers) will decide whether the new project should be added. The requester also needs to create an corresponding RFC for the change to happen, as described in [Decision Making and Voting](#decision-making-and-voting).
```

Comment:
```suggestion
New projects can be added to the TiKV organization via GitHub issue discussion in one of the existing projects in the organization, as long as as they adheres to the [CNCF charter](https://www.cncf.io/about/charter/) and the guidelines in this governance. Once sufficient discussions have taken place, the Maintainers of the project where the issue was opened (since different projects in the organization may have different maintainers) will decide whether the new project should be added. The requester also needs to create an corresponding RFC for the change to happen, as described in [Decision Making and Voting](#decision-making-and-voting).
```

There is no precedent for this, so we can't claim that.